src/Microsoft.Health.Fhir.SqlServer/Features/ChangeFeed/SqlServerFhirResourceChangeDataStore.cs

```diff
@@ -139,6 +141,11 @@ public async Task<IReadOnlyCollection<ResourceChangeData>> GetRecordsAsync(long
                     }
                 }
             }
+            catch (Exception ex) when ((ex is OperationCanceledException || ex is TaskCanceledException) && cancellationToken.IsCancellationRequested)
+            {
+                _logger.LogInformation(string.Format(Resources.GetRecordsAsyncOperationIsCanceled));
```

Comment:
nit: I think we need to include an ex object in the log information like the following.

_logger.LogInformation(ex, Resources.GetRecordsAsyncOperationIsCanceled);

Reply:
added